source/scss/brannan.scss

```diff
@@ -0,0 +1,45 @@
+/*
+ * Brannan
+ *
+ */
+@import 'shared';
+
+// mixin to extend lark filter
+// @mixin brannan
+// @param $filters... {filter} - Zero to many css filters to be added
+// @example
+//   img {
+//     @include brannan;
+//   }
+//   or
+//   img {
+//     @include brannan(blur(2px));
+//   }
+//   or
+//   img {
+//     @include brannan(blur(2px)) {
+//       /*...*/
+//     };
+//   }
+@mixin brannan($filters...) {
+  @extend %filter-base;
+  filter: sepia(.2) brightness(1.1) contrast(1.3) saturate(.8) $filters;
```

Comment:
This looks awesome! I was just tweaking the values a little bit and think this might work better: `filter: sepia(.25) brightness(1.05) contrast(1.2) saturate(.8) hue-rotate(-18deg) $filters;`